src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+@import 'header';
```

Comment:
Imported files should have an underscore prefix in their filenames to indicate that they are partials. Ensure that the actual filenames in the project directory start with an underscore, e.g., `_header.scss`, `_variables.scss`, etc.